ProcessLib/UncoupledProcessesTimeLoop.h

```diff
@@ -72,7 +72,8 @@ class UncoupledProcessesTimeLoop
     /// Maximum iterations of the global coupling.
     const unsigned _global_coupling_max_iterations;
     /// Convergence criteria of the global coupling iterations.
-    std::unique_ptr<NumLib::ConvergenceCriterion> _global_coupling_conv_crit;
+    std::vector<std::unique_ptr<NumLib::ConvergenceCriterion>>
```

Comment:
The docu needs an update about the vector.

Reply:
Added.